### PR TITLE
[#168024777] Enhancements

### PR DESCRIPTION
**Pull request by spleshka**

### CODE REVIEW

File: falconjs/packages/falcon/routing/xmlsitemap.js [hunk 1]
```diff
@@ -1,26 +1,32 @@
...
 
-  if (!xmlSitemapData) {
-    const superAgent = getRequest(app.nextConfig);
-    const sitemapResp = await superAgent
-      .get('/concern/sitemap.xml')
-      .set('Accept', 'application/xml');
+    if (!xmlSitemapData) {
+      const superAgent = getRequest(app.nextConfig);
+      const response = await superAgent
+        .get('/falcon/sitemap.xml')
```
Comment (PavelBulat) [inline falconjs/packages/falcon/routing/xmlsitemap.js:12]: @spleshka The file placed in falcon.js package. We should be able to configure url for sitemap (for another template it won't be ```falcon```) do you agree?

File: falconjs/packages/falcon/routing/xmlsitemap.js [hunk 2]
```diff
@@ -1,26 +1,32 @@
...
+        .set('Accept', 'application/xml');
 
-    xmlSitemapData = sitemapResp.body;
-    const cacheTTL = process.env.NODE_CACHE_TTL !== undefined
-      ? Number(process.env.NODE_CACHE_TTL)
-      : 1000;
-    cache.put('xmlSitemapData', xmlSitemapData, cacheTTL);
-  }
+      xmlSitemapData = response.body;
+      const cacheTTL = process.env.NODE_CACHE_TTL !== undefined
```
Comment (PavelBulat) [inline falconjs/packages/falcon/routing/xmlsitemap.js:16]: I believe it should be falcon config variable. (Not env variable)

File: falconjs/packages/falcon/utils/transforms.fields.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
-import he from 'he';
-import getEntityURL from './getEntityURL';
-import { BACKEND_URL } from './constants';
+const he = require('he');
```
Comment (PavelBulat) [inline falconjs/packages/falcon/utils/transforms.fields.js:1]: Why it is better than import in this case?

**Review by PavelBulat — CHANGES_REQUESTED** [review]
Great work! I added a couple of comments (check please)

File: falconjs/packages/falcon/routing/xmlsitemap.js (hunk 1 above)
Comment (spleshka) [reply falconjs/packages/falcon/routing/xmlsitemap.js:12]: @PavelBulat generally yes, but let's revisit it when we have second consumer (or at least 1 example with second consumer). That's totally fine as an example / starting point.

File: falconjs/packages/falcon/routing/xmlsitemap.js (hunk 2 above)
Comment (spleshka) [reply falconjs/packages/falcon/routing/xmlsitemap.js:16]: @PavelBulat I think yes, but again, you've used this env variable in multiple places already. Refactoring it here does not make too much sense to me - can be a follow-up if you want to fix it.

File: falconjs/packages/falcon/utils/transforms.fields.js (hunk 3 above)
Comment (spleshka) [reply falconjs/packages/falcon/utils/transforms.fields.js:1]: @PavelBulat because it has to work on the node.js level, imports do not work there.

**Review by PavelBulat — APPROVED** [review]

**Pull request merged**